github_deploy.admin.inc

```diff
@@ -80,7 +81,7 @@ function github_deploy_admin_settings() {
     '#description' => t('Method to secure Github deploy callback.'),
     '#options' => array(
       'secret_token' => t('Secret token'),
-      'whitelist_githubs_ip' => t("Whitelist GitHub's IP"),
+      'whitelist_githubs_ip' => t('Whitelist GitHub\'s IP'),
```

Comment:
I read it somewhere in drupal.org if you have to escape ' is better to use double quotation